tests/kuttl/tests/collapsed/01-assert-collapsed-cell.yaml

```diff
@@ -111,3 +104,57 @@ spec:
         service: CeilometerPassword
       secret: osp-secret
       serviceUser: ceilometer
+status:
```

Comment:
I'm seeing a bunch of other errors too.  Are the 3 tests running in parallel in CI?  That would be a problem I think.

Reply:
They should only run one at a time: https://github.com/openstack-k8s-operators/openstack-operator/blob/08801ea0fd19cfeebc2c6d6f9d520cc2e35397b6/kuttl-test.yaml#L23

Reply:
The problem is that because the test assertion fails, cleanup is not run, and then the following test cases can't deal with resources from previous deployment (rabbitmq seems particularly problematic)

Reply:
> Based on the results of the KUTTL test in CI, it looks like this list is out-of-date:
> 
> ```
> case.go:366: resource OpenStackControlPlane:openstack-kuttl-tests/openstack-collapsed-cell: .status.conditions: slice length mismatch: 13 != 18
> ```

I ran all tests locally manually, separated with `make crc_storage_cleanup` and `make crc_storage` and they all passed.